src/backend/AsyncMirror.ts

```diff
@@ -106,7 +107,7 @@ export default class AsyncMirror extends file_system.SynchronousFileSystem imple
                 if (err) {
                   cb(err);
                 } else if (i < files.length) {
-                  copyItem(`${p}/${files[i]}`, copyNextFile);
```

Comment:
Yes -- without it directories or files in the root directory give paths like `"//file"` (two slashes), and `copyItem` passes this directly to the bottom filesystem -- XHRFS in my tests.  XHRFS has names like `'/file'` in its map of files, not `'//file'`.  I could add a conditional here to check if `p[p.length-1] === '/'` to avoid calling join, although it looks somewhat reasonable on posix at least: https://github.com/nodejs/node/blob/master/lib/path.js#L1207